src/pocketmine/Player.php

```diff
@@ -1378,7 +1385,7 @@ protected function checkNearEntities($tickDiff){
 	}
 
 	protected function processMovement($tickDiff){
-		if(!$this->isAlive() or !$this->spawned or $this->newPosition === null or $this->teleportPosition !== null or $this->isSleeping()){
```

Comment:
That project is targeted at removing the irritating parts of the anti-cheat and placing them in plugins. However it makes zero sense to have a plugin for the sake of stopping dead players moving.